src/framework/_typography.scss

```diff
@@ -331,42 +331,13 @@ $font-sizes: (
 /*============================================================================*\
    Type font weight helpers
 \*============================================================================*/
+$font-weights: (300, 500, 800) !default;
```

Comment:
I would rather suggest the following default values : 

- 300: light
- 400: regular
- 700: bold

I think they are the most commonly used weights.